cpu/cc2538/radio/cc2538_rf.c

```diff
@@ -175,6 +175,7 @@ void cc2538_setup(cc2538_rf_t *dev)
     netdev_t *netdev = (netdev_t *)dev;
 
     netdev->driver = &cc2538_rf_driver;
+    dev->netdev.ops = &cc2538_ieee802154_ops;
```

Comment:
IMHO `cc2538_ieee802154_ops` and all its equivalents should be (and can be) `static const`? IIRC this also makes the calling of the function pointers more efficient, as with `static const` they are stored in ROM, with just `const` they are not.

Reply:
I will give it a look